rules/properties-alphabetical-order/tests/index.js

```diff
@@ -131,6 +146,13 @@ testRule({
 			fixed: '@media print { color: red; top: 0; }',
 			message: messages.expected('color', 'top'),
 		},
+		{
```

Comment:
We need more tests for `reject`. You could take the first case and add variation when the first property capitalized, and variation, where the second.